pages/apim/3.x/user-guide/publisher/promote-api.adoc

```diff
@@ -0,0 +1,60 @@
+= Promote APIs
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/apim_publisherguide_promote_apis.html
+:page-folder: apim/user-guide/publisher
+:page-layout: apim3x
+:page-liquid:
+
+== Overview
+
+The following sections describe how you can promote an API from an environment to another one.
```

Comment:
The following sections describe how you can promote an API from one environment to another.